ui/src/app/pages/access/modules-for-annual-renewal.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+
+import { AccessModule, Profile } from 'generated/fetch';
+
+import { FlexColumn } from 'app/components/flex';
+import { getAccessModuleStatusByNameOrEmpty } from 'app/utils/access-utils';
+
+import { RenewalCardBody } from './access-renewal';
+import { styles } from './data-access-requirements';
+
+interface RenewalCardProps {
+  profile: Profile;
+  modules: AccessModule[];
+}
+
+export const ModulesForAnnualRenewal = (props: RenewalCardProps) => {
```

Comment:
why was this broken out into its own file but not ModulesForInitialRegistration?

Reply:
I do not believe that this was intentional. I suspect it was an oversight on my part. I moved ModulesForInitialRegistration into its own file.